src/components/media/SongFrame.vue

```diff
@@ -23,14 +23,17 @@
         v-on="$listeners"
         @ready="ready"
       />
+      <Keypress @any="keyPress" />
```

Comment:
I think you could've just handled it by removing `disablekb:1` in line 16 of this file, instead of adding dependency.

Reply:
I actually wanted the keybinds then the YT embed is not in focus since I'm mainly interacting with the rest of the UI.

If the extra dep is not preferred, I can handle it manually with lifecycle hooks.

Reply:
@sphinxrave so... is this okay? and should it be in MusicBar2 instead?

Reply:
This is fine for a temporary solution